src/topology/builder.rs

```diff
@@ -287,9 +288,10 @@ pub async fn build_pieces(
                                 %error,
                                 component_kind = "sink",
                                 component_type = typetag,
-                                %component_id,
+                                component_id = ?component_id.id(),
```

Comment:
`component_id.id()` still seems weird, I'm guessing there's a reason we can't do `component.id()`

Reply:
🤔 I forgot this renaming from `component_id` to `component_key`...